js/Gameboard.js

```diff
@@ -1,23 +1,52 @@
-class Gameboard {
+  class Gameboard {
+    constructor() {
+      this.possibleCategories = this.generateCategories();
```

Comment:
`this.generateCategories` doesn't seem to be a method that you've defined on this class. You would be getting an error upon instantiating a new Gameboard with this code in its current state.